tests/rustdoc-ui/issue-79494.rs

```diff
@@ -1,5 +1,6 @@
 // only-x86_64-unknown-linux-gnu
+// check-pass
 
 #![feature(const_transmute)]
 
-const ZST: &[u8] = unsafe { std::mem::transmute(1usize) }; //~ ERROR cannot transmute between types of different sizes, or dependently-sized types
+const ZST: &[u8] = unsafe { std::mem::transmute(1usize) };
```

Comment:
Maybe make this public to force it to be evaluated? Otherwise #79494 could regress for documented items.